### PR TITLE
add pstake staked atom, stk/uatom

### PR DESCRIPTION
**Pull request by puneet2019**
add stkatom. 
https://www.mintscan.io/osmosis/txs/54F2C2BEF76F8FECEA17C3475EB8EFA49192AD14D5E9529CAC7E301EC77DF7CC
for the ibc txn
can search `ibc/CAA179E40F0266B0B29FB5EAA288FB9212E628822265D4141EBD1C47C3CBFCBC` in the raw JSON.

### CODE REVIEW

File: osmosis-1/osmosis-frontier.assetlist.json [hunk 1]
```diff
@@ -157,6 +157,34 @@
...
+    {
+      "description": "PSTAKE Liquid-Staked ATOM",
+      "denom_units": [
+        {
+          "denom": "ibc/CAA179E40F0266B0B29FB5EAA288FB9212E628822265D4141EBD1C47C3CBFCBC",
+          "exponent": 0,
+          "aliases": ["stk/uatom"]
+        },
+        {
+          "denom": "stk/atom",
```
[the remark this answers is not in the document]
Comment (puneet2019) [reply osmosis-1/osmosis-frontier.assetlist.json:169]: um, the denom on chain is stk/uatom, and I might add additional metadata as stk/atom on persistence chain itself

[the remark this answers is not in the document]
Comment (puneet2019) [reply osmosis-1/osmosis-frontier.assetlist.json:169]: i am adding this change, can always change it back, does osmosis-frontend use this?

**Review by IbrarMakaveli — APPROVED** [review]

**Comment by puneet2019** [comment]
@IbrarMakaveli @JeremyParish69 fixed conflicts
can i get this merged?  i want to add this to [frontier pool ](https://frontier.osmosis.zone/pool/843)

**Comment by IbrarMakaveli** [comment]
We are refactoring assetlist littlebit, going to merge after this PR is done : https://github.com/osmosis-labs/assetlists/pull/278
Can you also add "pools" base on that PR, Thanks

**Comment by puneet2019** [comment]
looks like we will do the pool later, 
have to create it with stable swap mechanism which is going to come in next release.. so its just the asset right now

File: osmosis-1/osmosis-1.assetlist.json [hunk 2]
```diff
@@ -174,6 +174,37 @@
...
+        "source_channel": "channel-6",
+        "dst_channel": "channel-4",
+        "source_denom": "stk/uatom"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/stkatom.png",
+        "svg": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/stkatom.svg"
+      },
+      "keywords": [
+        "osmosis-frontier"
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:205]: Without any pools defined, this keyword is not allowed. Please remove the "osmosis-frontier" keyword, or else define a pool.

File: osmosis-1/osmosis-1.assetlist.json [hunk 3]
```diff
@@ -174,6 +174,37 @@
...
+      "name": "PSTAKE staked ATOM",
+      "display": "stkatom",
+      "symbol": "stkATOM",
+      "ibc": {
+        "source_channel": "channel-6",
+        "dst_channel": "channel-4",
+        "source_denom": "stk/uatom"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/osmosis-labs/assetlists/main/images/stkatom.png",
```
Comment (JeremyParish69) [inline osmosis-1/osmosis-1.assetlist.json:201]: Please instead point these image URIs to the Cosmos Chain Registry. This repo will no longer be storing images here.

Comment (JeremyParish69) [reply osmosis-1/osmosis-1.assetlist.json:201]: "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkatom.png",
"svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/persistence/images/stkatom.svg"

Comment (puneet2019) [reply osmosis-1/osmosis-1.assetlist.json]: added, thanks!

File: osmosis-1/osmosis-1.assetlist.json (hunk 2 above)
Comment (puneet2019) [reply osmosis-1/osmosis-1.assetlist.json]: removed entire keywords, will add it back soon

**Pull request merged**